samples/generated/static-spa/public/app.js

```diff
@@ -203,7 +212,14 @@ function showSigninWidget() {
       authParams: {
         issuer: config.issuer,
         state: JSON.stringify(config.state),
-      }
+      },
+      idps: config.idps.split(/\s+/).map(v => {
+        const parts = v.split(/:/);
+        if (parts.length == 2) {
+          return {type: parts[0], id: parts[1]};
+        }
+        return null;
+      }).filter(v => v)
     });
```

Comment:
nit: (very nit). I like to keep the visual focus on the relevant parts of code to make it easier for the next coder who is skimming over the code and may not be familiar with it.  What do you think of this:

```
config.idps.split(/\s+/).map(idpToken => { 
    const [type, id] = idpToken.split(/:/);
    if(!type || !id) { 
       return null;
    }
    return { type, id};
}).filter(v => v);
```

Reply:
Done